src/lib/y2network/config.rb

```diff
@@ -109,7 +109,7 @@ def copy
     # @return [Boolean] true if both configurations are equal; false otherwise
     def ==(other)
       source == other.source &&
-        ((interfaces - other.interfaces) | (other.interfaces - interfaces)).empty? &&
+        interfaces == other.interfaces &&
```

Comment:
So the order matters?

Reply:
Oh, we replaced the arrays for `Y2Network::InterfacesCollection`. It looks good.